demos/storybook/stories/drawer/with-multiple-nav-groups.stories.ts

```diff
@@ -35,7 +35,7 @@ export const withMultiNavGroups = (): any => ({
               <pxb-spacer *ngIf="spacer"></pxb-spacer> 
               <pxb-drawer-nav-group [title]="groupTitle2" [divider]="true">
                  <pxb-drawer-nav-item *ngFor="let navItem of navItems2"
-                    [title]="navItem.title"
+                    [title]="state.open ? navItem.title : ''"
```

Comment:
Wait a second...shouldn't we be making this change in the component behavior?

Reply:
This story needs to updated to add icons to the second nav group.

Reply:
Yes. I think we also need to update the component too. The bug in question is 'labels getting chopped off when collapsed' - so shouldn't we update the component to hide the labels on collapse if there are no icons?